docs/pages/management/security/proxy-protocol.mdx

```diff
@@ -0,0 +1,80 @@
+  will parse the header and extract the client's IP address and port. If the header
+  isn't present, it will reject the connection with an error.
+- `off`: PROXY protocol is disabled. If a PROXY protocol header is received, it
+  will reject the connection with an error.
+- `unspecified` - PROXY protocol is allowed, but not required. If a PROXY protocol header is received,
+  it will replace the client' source IP address with the one from PROXY header, but source port will be set to `0`.
+  If the header isn't present, it will assume the IP address and port are
+  the same as the connection source IP address and port. **This is the default**
+
+Users are encouraged to explicitly set their `proxy_protocol` setting to `on` or `off` mode
+depending on the network setup. **Default unspecified value mode is not suitable for production**, it is only 
+suitable for test environments, in this mode Teleport doesn't not require PROXY prefix for the connection, but will 
+parse it if present. Incoming connections will also be marked by setting source port to `0` and this will be visible 
+in the audit events.
+
+<Admonition type="warning">
+
+IP pinning will not work if `proxy_protocol` setting wasn't explicitly set in the config. Connections that are marked 
+with source port = 0 will be rejected during IP pinning checks.
+
+</Admonition>
+
+Main rule for configuring `proxy_protocol` setting - **If Teleport is running behind a L4 load balancer, that is set to 
+send PROXY protocol headers, you should set `proxy_protocol: on`**.
+
+```yaml
+proxy_service:
+  proxy_protocol: on
+```
+
+Otherwise you should set `proxy_protocol: off`
+
+```yaml
+proxy_service:
+  proxy_protocol: off
+```
+
+You should make sure that Teleport is only accessible through load balancer, if you run it with `proxy_protocol: on`, so malicious 
+users can't spoof their IP address by connecting directly and sending custom PROXY header.
+Teleport only allows one PROXY protocol header for an incoming connection - that makes sure any client connecting through 
+PROXY enabled L4 load balancer will not be able to send their own PROXY header to the connection.
+
+PROXY protocol behavior is controlled separately for Auth service and Proxy service. So you should make sure both have correct 
+setting. If there's PROXY-enabled L4 load balancer between your Proxy and Auth services, you should set `proxy_protocol: on` on the Auth,
+otherwise set it to `off`.
+
+You can also have Teleport Proxies to have different PROXY protocol settings, if you run some of them behind load balancer and some not, for
+example if you have dedicated Proxies for private and public networks.
+
+Running Teleport behind L4 load balancer that doesn't send PROXY protocol headers will lead to all incoming connections seemingly
+coming from the same IP address from Teleport's point of view, making audit trail less useful and IP pinning feature not helpful.
```

Comment:
```suggestion
PROXY protocol behavior is controlled separately for Auth service and Proxy service. So you should ensure both have the correct 
setting. If there's a PROXY-enabled L4 load balancer between your Proxy and Auth services, you should set `proxy_protocol: on` on the Auth,
otherwise set it to `off`.

You can also have Teleport Proxies have different PROXY protocol settings, if you run some of them behind a load balancer and some not, for
example if you have dedicated Proxies for private and public networks.

Running Teleport behind an L4 load balancer that doesn't send PROXY protocol headers will lead to all incoming connections seemingly
coming from the same IP address from Teleport's point of view, making the audit trail less useful and the IP pinning feature not helpful.
```